week2/week2-homework

```diff
@@ -0,0 +1,117 @@
+# Week-2 homework
+# 1. Write fizz_buzz function
+from pywin.mfc.object import Object
+
+
+def fizz_buzz(number):
+    if number % 3 == 0 and number % 5 == 0:
+        return "FizzBuzz"
+    elif number % 3 == 0:
+        return 'Fizz'
+    elif number % 5 == 0:
+        return 'Buzz'
+    else:
+        return number
+
+
+print(fizz_buzz(12))
+
+# 2. write a try and except block to avoid IndexError
+lst = [5, 10, 20]
+try:
+    index = int(input('inter the index of an item in the list: > '))
+    element = lst[index]
+    print(element)
+except ValueError:
+    print('Index should be Integer number')
+except IndexError:
+    print('list index out of range')
+
+
+# 3. Create a class of Jet inventory with two arguments
+
+
+class Jet:
+    def __init__(self, name, country):
+        self.name = name
+        self.country = country
+        print(f'{name} {country}')
```

Comment:
I understand that the print should be in a method of that class, not in the constructor